topic6/ContulPersoanei.java

```diff
@@ -0,0 +1,71 @@
+package topic6;
```

Comment:
sugerez sa mai arunci un alt tip de exceptie, de ex IllegalArgumentException, sau NullPointerException in caz ca transmiti o valoare null unde este asteptat sa fie ceva, in sarrcina prarctica era specificat sa ai mai multe tipurri de exceptii, in rest Bravo!